src/main/resources/com/uber/jenkins/phabricator/HarbormasterConduitNotifier/global.jelly

```diff
@@ -0,0 +1,26 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <!--
+    This Jelly script is used to produce the global configuration option.
+
+    Jenkins uses a set of tag libraries to provide uniformity in forms.
+    To determine where this tag is defined, first check the namespace URI,
+    and then look under $JENKINS/views/. For example, <f:section> is defined
+    in $JENKINS/views/lib/form/section.jelly.
+
+    It's also often useful to just check other similar scripts to see what
+    tags they use. Views are always organized according to its owner class,
+    so it should be straightforward to find them.
+  -->
+  <f:section title="Phabricator">
+    <f:optionalBlock title="Harbormaster Conduit Publisher" name="harbormaster-conduit-notifier" checked="true">
+      <f:entry title="Conduit URL" field="conduitURL"
```

Comment:
Yea, I've been struggling with a better way to share this global configuration (since I'd like the conduitURL available to both pieces, and the conduit token is currently shared via an environment variable that could be maliciously accessed via a job). I will take a not to properly investigate better shared configuration.